content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -124,40 +128,48 @@ networkpolicy.networking.k8s.io/access-nginx created
 ```
 
 ## Test access to the service when access label is not defined
-If we attempt to access the nginx Service from a pod without the correct labels, the request will now time out:
+When you attempt to access the `nginx` Service from a Pod without the correct labels, the request times out:
 
 ```console
 kubectl run --generator=run-pod/v1 busybox --rm -ti --image=busybox -- /bin/sh
 ```
 
-```console
-Waiting for pod default/busybox-472357175-y0m47 to be running, status is Pending, pod ready: false
+```none
+If you don't see a command prompt, try pressing enter.
+```
 
-Hit enter for command prompt
+In your shell, run the following `wget` command:
 
+```console
 / # wget --spider --timeout=1 nginx
```

Comment:
This makes more sense as:
```suggestion
wget --spider --timeout=1 nginx
```

and in that case you can mark the block as `shell` rather than `console`.

Reply:
Done.